docs/guides/expectations/create_expectations_overview.md

```diff
@@ -3,4 +3,67 @@ title: "Create Expectations: Overview"
+- Completing [Step 3: Create Expectations](/docs/tutorials/getting_started/create_your_first_expectations) of the Getting Started tutorial is recommended.
+:::
+
+Creating Expectations is an integral part of Great Expectations.  By the end of this step, you will have created an Expectation Suite containing one or more Expectations which you will use when you Validate data.
+
+## The Create Expectations process
+
+There are a few workflows you can potentially follow when creating Expectations.  These workflows represent various ways of creating Expectations, although they converge in the end when you will save and test those Expectations.  Of the three potential ways to create Expectations, two are recommended in particular.
+
+First is the interactive work flow.  In this workflow, you will be working in a Python interpreter or Jupyter notebook.  You will use a Validator and its `expect()` method to define Expectations in an Expectation Suite, and when you have finished you will save that Expectation Suite into your Expectation Store.  A more thorough overview of this workflow, and a link to an in depth guide on it, can be found in this document's section on [creating Expectations interactively](#creating-expectations-automatically).
+
+The second workflow is the automated work flow.  In this workflow, you will use a Profiler to generate Expectations based on some input data.  The Validator will then save that those Expectations as an Expectation Suite in an Expectation Store.  You will also have the option of vieiwing them in your Data Docs.  A more thorough overview of this workflow, and a link to an in depth guide on it, can be found in this document's section on [creating Expectations automatically](#creating-expectations-automatically).
+
+The third workflow, which is not recommended, is to manually define your Expectations by writing their configurations.  This workflow does not require source data to work against, but does require a deep understanding of the configurations available for Expectations.  We will forgo discussion of it in this document, and focus on the two recommended workflows.  If for some reason you must use this workflow, we do provide an in depth guide to it in our documentation on [how to create and edit expectations based on domain knowledge without inspecting data directly](./how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md).
+
+Regardless of which of the first two workflows you have followed, once you have saved your Expectation Suite it is advised that you test them by validating them against the batch or batches of data you created them against.  This process is the same in either workflow, since at this point you will be using a saved Expectation Suite and each of the prior workflows ends with the saving of your Expectation Suite.  Instructions for this will be detailed in this document's section on [testing your Expectation Suite](#testing-your-expectation-suite).
```

Comment:
Since I recommend a slight change to the language about the third method above, it might make sense to change this as well.

```suggestion
When following one the first two workflows, once you have saved your Expectation Suite it is advised that you test it by validating your Expectations against the batch or batches of data against which you created them.  This process is the same in either workflow, since at this point you will be using a saved Expectation Suite and each of the prior workflows ends with the saving of your Expectation Suite.  Instructions for this will be detailed in this document's section on [testing your Expectation Suite](#testing-your-expectation-suite).
```